### ISSUE TITLE
[HW] Aadit

### ISSUE THREAD

**Issue by aadit-developer** [datastaxdevs/appdev-week3-graphql#321]
**Name:** Aadit
**Email:** <aaditpatel1@gmail.com>
**Linkedin Profile:** <https://www.linkedin.com/in/aadit-patel/>

Attach the homework screenshots below:
-----------------------------------------

<img width="1440" alt="Screenshot 2021-07-21 at 10 34 26 PM" src="https://user-images.githubusercontent.com/66415730/126530182-0f974aae-6b1d-409c-81f4-ead5426b60ee.png">

<img width="1440" alt="Screenshot 2021-07-21 at 10 34 50 PM" src="https://user-images.githubusercontent.com/66415730/126530064-620d9794-ee73-4ee2-b214-69fb0b8c76e1.png">
<img width="1440" alt="Screenshot 2021-07-21 at 10 40 21 PM" src="https://user-images.githubusercontent.com/66415730/126530602-d2b8c4f7-8d98-40ed-994d-cd949755662e.png">

<img width="1440" alt="Screenshot 2021-07-21 at 10 34 35 PM" src="https://user-images.githubusercontent.com/66415730/126529984-6e11f2f5-3e91-40d1-a8a9-016687520368.png">
<img width="1440" alt="Screenshot 2021-07-21 at 10 34 43 PM" src="https://user-images.githubusercontent.com/66415730/126530013-3af31958-87b5-446a-b5e8-fffade9fcf1e.png">

https://relaxed-einstein-b6a926.netlify.app/
<SCREENSHOTS>

**Comment by SonicDMG** [comment]
Hi Aadit, nice job getting the app deployed. I don't see your custom video though on your site. Could you please add one and take a screenshot or just post your netlify link. If you did add one please point out which one. Thank you.

**Comment by aadit-developer** [comment]
Hello,
So i need to add a custom movie in database right?

**Comment by SonicDMG** [comment]
That's correct. You can modify one of the graphQL mutations we used to insert the original 4 Sci-Fi movies per the workshop instructions. Just change the name, title, mp4, whatever.

**Comment by aadit-developer** [comment]
Will do, thanks!

**Comment by aadit-developer** [comment]
Hello,
I did the changes and added a movie star wars.
I have also attached the screenshot of the same.
The link of the deployed app is https://relaxed-einstein-b6a926.netlify.app
[image: Screenshot 2021-07-27 at 3.07.32 PM.png]

**Comment by HadesArchitect** [comment]
Hey **Aadit**, great job! Congrats! Here is your badge! https://api.badgr.io/public/assertions/OLd6dL_eSuGuAPUlU3T1ZA
